lib/usdc.ts

```diff
@@ -50,7 +48,6 @@ export const executeUSDCTransfer = async ({
 }) => {
```

Comment:
In the JSdoc: it should be coordinates Instead of publicKey.

Reply:
Updated in the other pending PR